frontend/src/components/pages/AdminDashboard/AdminDashboard.tsx

```diff
@@ -215,10 +215,13 @@ const AdminDashboard = (): React.ReactElement => {
     if (data.length > 0) {
       data.forEach((review: ReviewResponse) => {
         const names: string[] = [];
-        if (review.books[0].seriesName === null) {
+        if (
+          review.books[0].series.name === null ||
```

Comment:
In the case of only 1 book in the review, we should display the Book's title, not the series title. (Just realized this wasn't the case even before)
In the case of multiple books, we should display the series title of the 0th book